### ISSUE TITLE
React doesn't load with version v6.648?

### ISSUE THREAD

**Issue by AZFire1** [VolmitSoftware/React#587]
**Describe the bug**
Since the new version, the plugin does not load?

**To Reproduce**
1. Load the new version of the plugin

**Expected behavior**
The plugin to load.

**Server and Plugin Informations**
 - Installed plugins: https://bin.bloom.host/odeveqorop.m
 - React Version: v6.648
 - Server Platform and Version [eg: PaperSpigot 1.14.4]: Purpur
 - Operating System (if applicable): Not applicable

**Additional context**
https://bin.bloom.host/ufuhazunal.sql invalid plugin.yml error

**Comment by sanderdatema** [comment]
Yep, can confirm. Although the error is `Jar does not contain plugin.yml`.

```
[22:19:37 ERROR]: Could not load 'plugins/React-6.648.jar' in folder 'plugins'
org.bukkit.plugin.InvalidDescriptionException: Invalid plugin.yml
        at org.bukkit.plugin.java.JavaPluginLoader.getPluginDescription(JavaPluginLoader.java:162) ~[patched_1.16.5.jar:git-Purpur-978]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:148) ~[patched_1.16.5.jar:git-Purpur-978]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.loadPlugins(CraftServer.java:388) ~[patched_1.16.5.jar:git-Purpur-978]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:216) ~[patched_1.16.5.jar:git-Purpur-978]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:942) ~[patched_1.16.5.jar:git-Purpur-978]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:176) ~[patched_1.16.5.jar:git-Purpur-978]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.io.FileNotFoundException: Jar does not contain plugin.yml
        ... 7 more
```

**Comment by relavis** [comment]
Fixed in 6.650

**Comment by AZFire1** [comment]
My bad, got it wrong.
